playbooks/robusta_playbooks/pod_evicted_enrichments.py

```diff
@@ -0,0 +1,80 @@
+import logging
+from typing import List
+
+from hikaru.model.rel_1_26 import Node, PodList
+from robusta_playbooks.playbook_utils import pod_row
+
+from robusta.api import (
+    BaseBlock,
+    EnrichmentType,
+    Finding,
+    FindingSeverity,
+    PodEvent,
+    PodFindingSubject,
+    TableBlock,
+    action,
+)
+
+
+@action
+def on_pod_evicted_enricher(event: PodEvent):
+    """
+    Retrieves pod and node information for an OOMKilled pod
+    """
+    pod = event.get_pod()
+    if not pod:
+        logging.error(f"cannot run on_pod_evicted_enricher on event with no pod: {event}")
+        return
+
+    try:
+        node = Node.readNode(pod.spec.nodeName).obj
```

Comment:
Lets move this common node logic to a node util file

Reply:
Moved to  RobustaPod class